packages/webpack/src/buildtime/assemble.js

```diff
@@ -43,7 +43,9 @@ const assembleRuntime = (KEY, runtimeModules) => {
         LAVAMOAT['${name}'] = module.exports;
       })();`
     }
-    assembly += `\n;/*${name}*/;\n${sourceString}`
+    if (sourceString) {
```

Comment:
This is clever, but I don't like it for one reason - passing a broken item to asseble is now a silent error

Reply:
For sourceString to be updated, it takes only one of `{ file, data, name, json, shimRequire }` to be truthy, which means that such filter won't cut it, and instead we'll have to test all of them in order to filter stuff out.

Essentially, the filter suggestion is no different than the current one in context of "silent error" - filtering out broken items without letting anyone know about it is essentially the same in both scenarios, I just chose the one scenario that requires the smaller amount of checks, which is the final sink of whether the item caused a string change or not.